server/controllers/cohort.js

```diff
@@ -159,17 +159,22 @@ const getCohortDetails = (req, resp) => {
       if (isArchived) {
         return resp.status(200).json({ _id, isArchived, name });
       }
-
+      const arrayOfMemberIds = membersCollection.map(member => member._id);
```

Comment:
Is this check really necessary? In line 139 we look for a cohort where "memberIds" contains current user id (we should remove check for the "ownerIds" in the future).  If the cohort was found, we know that the user is a member for sure and we don't have to check this twice (extract members' ids from membersColleciton and iterate over the "arrayOfMemberIds". If you need to set isMember prop before sending cohort details to the user (because this prop is necessary in frontend), you can set it directly to true. Is there any additional reason to leave this check?